ci-operator/step-registry/ovn/utils/ovn-utils-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: ovn-utils
+  from_image:
+    name: cli-yq
```

Comment:
not sure it matters, but it could use a smaller/simpler from_image, since it just needs `bash` and `cat`

Reply:
I don't need `yq` at the moment, but in my latest update to this PR I'm using `oc` so I think I need at least cli. That's what I have it at now.

Reply:
actually, just `cli` didn't work. going back to `cli-yq`